package.json

```diff
@@ -21,8 +21,8 @@
         "format": "prettier -c . && yarn workspace semaphore-docs format",
         "format:write": "prettier -w . && yarn workspace semaphore-docs format:write",
         "docs": "typedoc",
-        "version:bump": "yarn workspaces foreach -A --no-private version -d ${0} && yarn version apply --all && yarn remove:stable-version-field && NO_HOOK=1 git commit -am \"chore: v${0}\" && git tag v${0}",
-        "version:publish": "yarn build:libraries && yarn clean:cli-templates && yarn workspaces foreach -A --no-private npm publish --tolerate-republish --access public",
+        "version:bump": "ts-node scripts/version.ts ${0}",
```

Comment:
no directly related to this PR, but given the length of this command, I suggest extracting it in its own script definition file to improve readability